### ISSUE TITLE
Hey Riley, this is one of your beta testers running the copy of Delta beta 5 you sent us earlier on in the month. I've sideloaded the .ipa using Ext3nder on my jailbroken iPhone 7 Plus and restored my database folder. All of my games show up normally however upon booting up any game I get no sound. I've checked my ringer settings and flipped the switch on and off and nothing. Turned up the volume obviously still nothing. Tried plugging in headphones to see if it's just a bug with speaker output but no sound comes out of those either. Pretty strange. Any help would be appreciated. Thx!

### ISSUE THREAD

**Issue by 11SoniCHunter87** [rileytestut/Delta#76]
Hey Riley, this is one of your beta testers running the copy of Delta beta 5 you sent us earlier on in the month. I've sideloaded the .ipa using Ext3nder on my jailbroken iPhone 7 Plus and restored my database folder. All of my games show up normally however upon booting up any game I get no sound. I've checked my ringer settings and flipped the switch on and off and nothing. Turned up the volume obviously still nothing. Tried plugging in headphones to see if it's just a bug with speaker output but no sound comes out of those either. Pretty strange. Any help would be appreciated. Thx!

__Originally posted by @mk8itra1n in https://github.com/rileytestut/Delta/issues/2__

**Comment by 11SoniCHunter87** [comment]
Install

**Comment by Assfugil** [comment]
uhh please don't put everything in the title, that's what description is for.

**Comment by GeonicTREE** [comment]
Has anyone solved this problem yet?

**Comment by DX12development** [comment]
> Has anyone solved this problem yet?
> 
> I'm using an iPhone 7. And it's audio calls and microphone is broken. I can't hear and answer calls, it is a pretty known problem on iPhone 7. It happened when I drop the phone, which results a broken home button and audio. But speakers is fine, I can hear sounds just not from calling.
> 
> I have no idea why. But I just think this so the reason why I can't hear audio. Though i tried iNDS and there is no sound problem. Any idea why?
> 
> And I tried everything already. I donwloaded through altstore or with an ipa file but still nothing. And no my silent mode isn't turned on.

I have the same issue. Same phone and mic problem. I've tried using a ~1.2 beta version before the official release of the DS emulator and that worked fine. I got sound, but the performance was a lot worse than the public release. The beta version I tried had no microphone option, which makes me think this issue has to do with delta trying to access the microphone and messing up cause the mic on the broken iphone 7+ causes a lot of apps to have problems starting (like garageband which always has the microphone on in the background).

**Comment by shanegillio** [comment]
Hi there! Are you still experiencing issues in the latest version, 1.4?

**Comment by jach26** [comment]
> Hi there! Are you still experiencing issues in the latest version, 1.4?

yes, a year later. It could be a problem with the microphone option, so a toggle would be nice. And also, a toggle for the Ds's audio itself would be amazing.
